src/main/java/net/snowflake/ingest/utils/Constants.java

```diff
@@ -24,6 +24,10 @@ public class Constants {
   public static final String PRIVATE_KEY_PASSPHRASE = "private_key_passphrase";
   public static final String JDBC_PRIVATE_KEY = "privateKey";
   public static final long RESPONSE_SUCCESS = 0L; // Don't change, should match server side
+  public static final long RESPONSE_ERR_GENERAL_EXCEPTION_RETRY_REQUEST =
+      10L; // Don't change, should match server side
+  public static final long RESPONSE_ERR_ENQUEUE_TABLE_CHUNK_QUEUE_FULL =
+      7L; // Don't change, should match server side
```

Comment:
Could you add a comment at server side indicating that this error code can't be changed, same thing as what I did in https://github.com/snowflakedb/snowflake/commit/478cbfc305f41f7074bcd52aad5dfd62a7fb96e9